docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-product-options-feature-walkthrough.md

```diff
@@ -13,9 +12,26 @@ To learn more about the feature and to find out how end users use it, see [Marke
 
 {% endinfo_block %}
 
-## Related Developer articles
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/d8882366-b2dd-4d6c-b401-01db47a00481.png?utm_medium=live&utm_source=custom)
```

Comment:
1. MerchantProductOptionDataImport does not have a dependency to MerchantProductOptionStorage
2. If the module is part of the feature it should be blue.
3. Please avoid arrows overlap.